lua/gatekeeperc.lua

```diff
@@ -0,0 +1,78 @@
+local ffi = require("ffi")
+
+--Structs
+ffi.cdef[[
+
+struct bp_config {
```

Comment:
All this duplicated code in Lua and C is a trouble to maintain. How are other projects using LuaJIT dealing with this issue?